telegram/ext/conversationhandler.py

```diff
@@ -76,6 +76,10 @@ class ConversationHandler(Handler):
         per_user (:obj:`bool`): Optional. If the conversationkey should contain the User's ID.
         per_message (:obj:`bool`): Optional. If the conversationkey should contain the Message's
             ID.
+        name (:obj:`str`): Optional. The name for this conversationhandler. Required for
+            persistence
+        persistent (:obj:`bool`): Optional. If the conversations dict for this handler should be
```

Comment:
If name is required, could we not just have a single `persistent_name` that turns persistence on if a string?

Reply:
Interesting. But this way we also have the name seperate if not persistent. Might be usefull some day?